kobuki_safety_controller/package.xml

```diff
@@ -1,12 +1,12 @@
-<package>
+<package format="2">
```

Comment:
Shouldn't we be leading with the following these days?

```
<?xml version="1.0" encoding="utf-8"?>
<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
<package format="3">
```

Reply:
Mostly fixed in 9b22898.  While we can definitely switch to format 3 if you like, the general consensus is that we only need to switch to format 3 if we need one of the features of it.  Since this package doesn't have any messages/services/actions, or the need for conditional dependencies, I didn't switch to it.  I don't feel strongly either way.

Reply:
Understood.